src/components/layout/layout.js

```diff
@@ -16,7 +16,7 @@ import { DerivStore } from 'store'
 const LiveChat = Loadable(() => import('./livechat'))
 
 const Main = styled.main`
-    padding-top: ${(props) => props.padding_top || '7rem'};
+    top: ${(props) => props.top || '7rem'};
```

Comment:
can you help to explain why?

Reply:
Padding creates white space which will be covered by nav bar. it will show only when you scroll down on ios devices. Since Hero section needs have a space between the nav and content. `top` property appears to be a correct solution.

Reply:
> `Padding` creates white space which will be covered by nav bar. it will show only when you scroll down on ios devices. Since Hero section needs have a space between the nav and content. `top` property appears to be a correct solution.

`fixed in the latter commit`